### PR TITLE
remove google kickstart and hackerrank

### LINKED ISSUE #4053: Remove Inactive Contest Sites
https://usaco.guide/general/contests

Some of these should be removed (e.g., hackerrank)

### PR DESCRIPTION
**Pull request by devo1ution**
fixes #4053
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [x] I have linked this PR to any issues that it closes.

### CODE REVIEW

File: content/1_General/Contests.mdx [hunk 1]
```diff
@@ -85,7 +76,7 @@ Let me know if there's something else I should try to qualify for!
 	<Resource
 		source="TopCoder"
 		title="Open"
-		url="https://tco20.topcoder.com/competition-overview/algorithm/algorithm-ways"
+		url="https://www.topcoder.com/community/member-programs/topcoder-open"
```
Comment (bqi343) [inline content/1_General/Contests.mdx:79]: can you remove this one too? no longer held

**Review by bqi343 — CHANGES_REQUESTED** [review]

File: content/1_General/Contests.mdx (hunk 1 above)
Comment (devo1ution) [reply content/1_General/Contests.mdx:79]: really? is this topcoder open 2023 not the same thing https://tco23.topcoder.com/

Comment (bqi343) [reply content/1_General/Contests.mdx:79]: https://codeforces.com/blog/entry/116814

Comment (devo1ution) [reply content/1_General/Contests.mdx:79]: ah ok

**Review by bqi343 — APPROVED** [review]

**Pull request merged**